app/experimenter/settings.py

```diff
@@ -173,7 +175,7 @@
     "loggers": {
         "django.db": {
             "handlers": ["console"] if DEBUG else [],
-            "level": "DEBUG",
+            "level": "WARNING",
```

Comment:
@peterbe Added a separate PR for making this more easily configurable, so you don't need to do this here.